aim/config.py

```diff
@@ -32,18 +32,6 @@
 
 LOG = logging.getLogger(__name__)
 
-global_opts = [
```

Comment:
Neutron already provides a "host" global option, this was only registered in cases where the standard Neutron config wasn't available (eg. the CLI or AIM UTs)